sdk/cosmos/azure-cosmos/azure/cosmos/aio/database.py

```diff
@@ -230,7 +230,9 @@ async def create_container(
         if response_hook:
             response_hook(self.client_connection.last_response_headers, data)
 
-        return ContainerProxy(self.client_connection, self.database_link, data["id"], properties=data)
+        container = ContainerProxy(self.client_connection, self.database_link, data["id"], properties=data)
+        await container._check_is_system_key()
```

Comment:
this is the big change, where we preemptively do this network call before sending back the container instance to the user

Reply:
At @simorenoh do you have any concerns with this approach? This call is already awaiting other async calls, so this doesn't strike me as crazy. @annatisch?

Reply:
Depends on what the user is doing, but so long as they're passing in a partition key value to any of the functions affected (read_item(), query_items(), delete_item(), query_conflicts(), read_conflict(), delete_conflict()) they will have to use this result.

Since the list above includes several item methods, I'd say there's a good chance they'd need it.

Reply:
Nevermind, there is more things to think about... a proper concern with this in that users may not necessarily create containers (and instead use `get_container_client()` to obtain their existing instance) which would still not work, will think about this more.